cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -136,8 +142,9 @@ public Package getPackage(String orgNamePath, String packageNamePath, String ver
                     }
     
                     // If request sent is wrong or error occurred at remote repository
-                    if (getPackageResponse.code() == HttpsURLConnection.HTTP_BAD_REQUEST ||
-                        getPackageResponse.code() == HttpsURLConnection.HTTP_INTERNAL_ERROR) {
+                    if (getPackageResponse.code() == HTTP_BAD_REQUEST ||
```

Comment:
Instead of throwing the same exception for all errors will it be better to throw different errors for different error ranges. E.g. one for http client errors (code: 400-499) and another for http server errors (500-599). I think when troubleshooting, this will make it easier to identify where the issue lies. @hemikak @hevayo WDYT?

Reply:
In central API implementation level, we usually return the same error structure but with different error messages. The error message should be sufficient enough to identify the underlying cause.